assignments/arrays.js

```diff
@@ -2,95 +2,185 @@
+  { id: 35, car_make: "Chrysler", car_model: "300M", car_year: 2000 },
+  { id: 36, car_make: "Volvo", car_model: "XC70", car_year: 2003 },
+  { id: 37, car_make: "Oldsmobile", car_model: "LSS", car_year: 1997 },
+  { id: 38, car_make: "Toyota", car_model: "Camry", car_year: 1992 },
+  { id: 39, car_make: "Ford", car_model: "Econoline E250", car_year: 1998 },
+  { id: 40, car_make: "Lotus", car_model: "Evora", car_year: 2012 },
+  { id: 41, car_make: "Ford", car_model: "Mustang", car_year: 1965 },
+  { id: 42, car_make: "GMC", car_model: "Yukon", car_year: 1996 },
+  { id: 43, car_make: "Mercedes-Benz", car_model: "R-Class", car_year: 2009 },
+  { id: 44, car_make: "Audi", car_model: "Q7", car_year: 2012 },
+  { id: 45, car_make: "Audi", car_model: "TT", car_year: 2008 },
+  { id: 46, car_make: "Oldsmobile", car_model: "Ciera", car_year: 1995 },
+  { id: 47, car_make: "Volkswagen", car_model: "Jetta", car_year: 2007 },
+  { id: 48, car_make: "Dodge", car_model: "Magnum", car_year: 2008 },
+  { id: 49, car_make: "Chrysler", car_model: "Sebring", car_year: 1996 },
+  { id: 50, car_make: "Lincoln", car_model: "Town Car", car_year: 1999 }
+];
 
 // PROJECT RESTRICTION: You can't use map, reduce, or filter to solve these problems.  Only use native JavaScript for loops.
-
+// !!! JOSH REMOVED THESE RESTRICTIONS IN WEB17 CHAT !!!
 // Example for loop:
 // arr = [1,2,3,4];
 // for (i = 0; i < arr.length; i++) {
 //     arr[i]; // 1,2,3,4
 // }
 
 // ==== Challenge 1 ====
-// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
-
+// The dealer can't recall the information for a car with an id of 33 on his lot.
+// Help the dealer find out which car has an id of 33 by logging the car's year,
+// make, and model in the console log provided to you below:
+const car = findCarID(33);
+function findCarID(carID) {
+  for (let i = 0; i < inventory.length; i++) {
+    const car = inventory[i];
+    if (car.id === carID) {
+      return car;
+    }
+  }
+}
+console.log(
+  `Challenge 1: Car ${car.id} is a ${car.car_year} ${car.car_make} ${
+    car.car_model
+  }`
+);
 
 // ==== Challenge 2 ====
-// The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+// The dealer needs the information on the last car in their inventory.
+// What is the make and model of the last car in the inventory?  Log the make and model into the console.
+const lastCar = inventory[inventory.length - 1];
+console.log(
+  `Challenge 2: Car ${lastCar.id} is a ${lastCar.car_make} ${lastCar.car_model}`
+);
 
 // ==== Challenge 3 ====
```

Comment:
good job referencing previously written code. Don't reinvent the wheel if you don't have to. Thinking like a programmer!